### ISSUE TITLE
Do we still need to explicitly record units in the highlighter record calls?

### ISSUE THREAD

**Issue by robintw** [debrief/pepys-import#226]
At the moment, whenever we record an extraction for use with the highlighter, we do something like this:

```python
heading_token.record(self.name, "heading", heading, "degrees")
```

However, now that the `heading` variable is a Quantity which contains a float value and a unit, the string conversion of this variable automatically includes a unit. For example, in this case `str(heading)` would give `238 degree`.

Therefore, do we need to include the explicit unit argument (the final one)?

**Comment by IanMayo** [comment]
You're right - in this case the value of `heading` is _self-describing_.  So, we can leave the units empty/None.   You're welcome to make the fix.

I don't know that all values will be self-describing.  I'm sure the  `LogsHoldings` will have bespoke units (_bushels_ :-) ).

So, I think we'll need to keep the parameter.

**Comment by robintw** [comment]
Fixed in #235
